ocw/lib/emailnotify.py

```diff
@@ -65,26 +64,24 @@ def send_leftover_notification():
 
 
 def send_cluster_notification(namespace, clusters):
-    cfg = ConfigFile()
-    cfg_path = ['notify.cluster.namespace.{}'.format(namespace), 'to']
-    if not cfg.has('notify') or not cfg.has(cfg_path):
+    config_path = 'notify.cluster.namespace.{}/to'.format(namespace)
+    if not ConfigFile().has('notify') or not ConfigFile().has(config_path):
         return
     if len(clusters):
         clusters_str = ' '.join([str(cluster) for cluster in clusters])
         logger.debug("Full clusters list - %s", clusters_str)
-        send_mail("EC2 clusters found", clusters_str, receiver_email=cfg.get(cfg_path))
+        send_mail("EC2 clusters found", clusters_str, receiver_email=ConfigFile().get(config_path))
 
 
 def send_mail(subject, message, receiver_email=None):
-    cfg = ConfigFile()
-    if not cfg.has('notify'):
+    if not ConfigFile().has('notify'):
         return
 
-    smtp_server = cfg.get(['notify', 'smtp'])
-    port = cfg.get(['notify', 'smtp-port'], 25)
-    sender_email = cfg.get(['notify', 'from'])
+    smtp_server = ConfigFile().get('notify/smtp')
```

Comment:
```suggestion
    smtp_server = PCWConfig.get_feature_property('notify/smtp')
```
All these `get()` should be moved to `get_feature_property()`.

Reply:
done